server.js

```diff
@@ -4826,13 +4833,16 @@ cache(function(data, match, sendBadge, request) {
         sendBadge(format, badgeData);
       } else if (info === 'l') {
         var license = data.license;
-        if (license === '') {
-          badgeData.text[1] = 'Unknown';
-        } else {
-          badgeData.text[1] = license;
+        if (license) {
```

Comment:
I'm not a big fan of implicit conversion to booleans in if / for / while statements, given that they can get counter-intuitive. Could you convert it to a check for a list, along with a check for the existence of at least one license?